### ISSUE TITLE
Cannot read property 'states' of undefined

### ISSUE THREAD

**Issue by matt8707** [TheLastProject/lovelace-valetudo-map-card#45]
Ha 0.110 breaks valetudo-map-card when placed in picture elements.

```
hui-weather-forecast-card.ts:39 hui-picture-elements-card TypeError: Cannot read property 'states' of undefined
    at HTMLElement.set hass [as hass] (valetudo-map-card.js:304)
    at ha-state-label-badge.ts:24
    at Array.map (<anonymous>)
    at HTMLElement.value (ha-state-label-badge.ts:24)
    at s (hui-weather-forecast-card.ts:39)
    at hui-weather-forecast-card.ts:39
    at l (hui-weather-forecast-card.ts:39)
    at Module.Ht (hui-weather-forecast-card.ts:39)
    at u (hui-element.js:1)
    at HTMLElement.setConfig (hui-element.js:1)
```

**Comment by TheLastProject** [comment]
Thanks for figuring out the issue lies with the combination with picture cardm Could you paste your card's config to help my testing, so I'm sure I'm looking at the same thing?

**Comment by matt8707** [comment]
It's the commented out part of[ this file ](https://github.com/matt8707/hass-config/blob/master/popup/sidebar_vacuum.yaml)  but I can reproduce with your example

```
type: picture-elements
image: https://online.visual-paradigm.com/repository/images/e5728e49-09ce-4c95-b83c-482deee24386.png
elements:
  - type: 'custom:valetudo-map-card'
    entity: sensor.xiaomi_map
    show_floor: false
    background_color: transparent
```

**Comment by TheLastProject** [comment]
Fixed in 42dbceb3b75c591fe0567a9e78657f9695daf82a
